crates/rune-macros/src/any.rs

```diff
@@ -657,7 +657,7 @@ where
                 impl #impl_generics #unsafe_to_ref for #ident #type_generics #where_clause {
                     type Guard = #raw_into_ref;
 
-                    unsafe fn unsafe_to_ref<'a>(value: #value) -> #vm_result<(&'a Self, Self::Guard)> {
+                    unsafe fn unsafe_to_ref<'__rune_g>(value: #value) -> #vm_result<(&'__rune_g Self, Self::Guard)> {
```

Comment:
This wasn't strictly necessary, but I noticed that the lifetime here conflicts with the lifetime specified in the struct. Since `'a` is a very common lifetime, I renamed it to an "internal" lifetime to make it less likely to conflict.